lib/manager/jenkins/extract.ts

```diff
@@ -0,0 +1,36 @@
+import * as datasourceJenkins from '../../datasource/jenkins';
+import { logger } from '../../logger';
+import * as dockerVersioning from '../../versioning/docker';
+import { PackageDependency, PackageFile } from '../common';
+
+export function extractPackageFile(content: string): PackageFile | null {
+  logger.trace('jenkins.extractPackageFile()');
+  const deps: PackageDependency[] = [];
+
+  for (const line of content.split('\n')) {
+    const match = /^\s*([\d\w-]+):([^#\s]+).*$/.exec(line);
```

Comment:
```suggestion
    const match = /^\s*(?<depName>[\d\w-]+):(?<currentValue>[^#\s]+).*$/.exec(line);
```

Please also move this regex into the top-level scope so that it's reused and not recreated each for loop.

Reply:
Good point, fixed this.